pool/pool.go

```diff
@@ -395,6 +748,28 @@ func (x *PrmBalanceGet) SetOwnerID(ownerID owner.ID) {
 	x.ownerID = ownerID
 }
 
+type PrmCreateSession struct {
```

Comment:
Can it be private? I don't see any usage in exported method.

Same for `PrmEndpointInfo`, `PrmNetworkInfo` and `ResCreateSession`.